public/app/features/storage/ExportView.tsx

```diff
@@ -78,6 +77,13 @@ export const ExportView = ({ onPathChange }: Props) => {
     getBackendSrv().post('/api/admin/export/stop');
   };
 
+  const setInclude = useCallback(
+    (k: string, v: boolean) => {
+      setBody({ ...rawBody!, include: { ...rawBody!.include, [k]: v } });
```

Comment:
This is pretty cool effect (switching toggle updates include json property in real time) - though I guess that is what we are using as a payload to endpoint so makes sense